release-content/0.14/release-notes/12889_Implement_GPU_frustum_culling.md

```diff
@@ -1 +1,34 @@
-TODO
+<!-- GPU Frustum Culling: https://github.com/bevyengine/bevy/pull/12889-->
+
+Frustum culling can now optionally be performed on the GPU.
+
+To enable it, add the `GpuCulling` component to the camera.
+
+```rust
+commands.spawn((
+    Camera3dBundle::default(),
+    GpuCulling
+));
+```
+
+To disable CPU frustum culling, you'll need to add the `NoCpuCulling`
+component to the camera.
+
+```rust
+commands.spawn((
+    Camera3dBundle::default(),
+    NoCpuCulling
+));
+```
+
+**Note:** `GpuCulling` does not imply `NoCpuCulling`. To only perform frustum
+culling on the GPU, you need to add both `GpuCulling` and `NoCpuCulling` to the
+camera.
+
+```rust
```

Comment:
I think we can condense this section substantially, by only including this snippet and then leaving a comment above each line as needed.

Reply:
Re-worked to just have one example with comments.